components/textfield/stories/textarea.stories.js

```diff
@@ -0,0 +1,144 @@
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import metadata from "../metadata/metadata.json";
+import packageJson from "../package.json";
+import { HelpTextOptionsTextArea, Template } from "./textarea.template.js";
+import { TextAreaGroup } from "./textarea.test.js";
+import { default as Textfield } from "./textfield.stories.js";
+
+/**
+ * A text area is multi-line text field using the `<textarea>` element that lets a user input a longer amount of text than a standard text field. It can include all of the standard validation options supported by the text field component.
+*/
+
+export default {
+	title: "Text area",
+	component: "TextArea",
+	argTypes: {
+		...Textfield.argTypes
+	},
+	args: {
+		...Textfield.args,
+		labelText: "Comments"
+	},
+	parameters: {
+		packageJson,
+		metadata,
+	}
+};
+
+export const Default = TextAreaGroup.bind({});
+Default.args = {};
+
+// ********* DOCS ONLY ********* //
+
+export const CharacterCount = Template.bind({});
+CharacterCount.tags = ["!dev"];
+CharacterCount.args = {
+	labelText: "Comments",
+	displayCounter: true,
+	characterCount: 50,
+	value: "Duis mollit ut laboris est labore sunt ipsum. Proident nostrud in ea reprehenderit proident nostrud. Anim ut est anim ex amet."
+};
+CharacterCount.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * A text area in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+*/
+export const Disabled = Template.bind({});
```

Comment:
When you run this, is the `Disabled` arg showing up in a "Tag settings" section in the control table? I only see that in text area, but maybe it's just me. 🤞 

<img width="158" alt="Screenshot 2024-10-18 at 10 21 58 AM" src="https://github.com/user-attachments/assets/618e8006-935e-4f56-bd6f-6e6e4aba01f3">

Reply:
@marissahuysentruyt Not sure why it shows up like that, but it doesn't appear like that for me.